e2etest/bgp_test.go

```diff
@@ -534,6 +534,101 @@ var _ = ginkgo.Describe("BGP", func() {
 				return nil
 			}, 4*time.Minute, 1*time.Second).Should(BeNil())
 
+			ginkgo.By("checking metrics")
```

Comment:
I think the metrics tests should be a different describetable/it but under the "BFD" context.

Reply:
+1
The name must contain BFD or these tests will run in the native lanes too

Reply:
Ah just saw this is under BFD context.